src-app/biz-modules/user-settings/pages/account-page.tsx

```diff
@@ -0,0 +1,133 @@
+import { Account } from '@app/biz-modules/user-settings/api'
+import { AppLayout } from '@app/layouts/app-layout'
+import { ErrorSummary, PageHeader, SuccessNotice } from '@app/library/release'
+import { Button } from '@ds/release'
+import { useCallback, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { DataField, Field } from '../components/data-field'
+import { ACCOUNT_EMPTY, useUserAccount } from '../state'
+
+type AccountRecord = Partial<Record<keyof Account, string>>
+
+const AccountPage = () => {
+	useTranslation()
+	const { account, accountLoading, updateAccount } = useUserAccount()
+	const [payload, setPayload] = useState<Account>(ACCOUNT_EMPTY)
+	const [feedback, setFeedback] = useState<AccountRecord>(ACCOUNT_EMPTY)
+	const [successful, setSuccessful] = useState(false)
+
+	const legendClass = cx('mb-sm-3 text-size-lg text-color-text-subtle')
+
+	const publicFields: Field<keyof Account>[] = [
+		{ key: 'name', label: t('userAccount.label.name') },
+		{ key: 'avatar', label: t('userAccount.label.avatar') },
+	]
+
+	const privateFields: Field<keyof Account>[] = [
+		{ key: 'email', label: t('userAccount.label.email') },
+		{ key: 'phone', label: t('userAccount.label.phone'), optional: true },
+	]
+
+	const canSave =
+		account.name !== payload.name.trim() ||
+		account.avatar !== payload.avatar.trim() ||
+		account.email !== payload.email.trim() ||
+		account.phone !== payload.phone.trim()
+
+	const hasErrors = (errors: object) => Object.values(errors).some((value: string) => value)
+
+	const onSubmit = useCallback(async () => {
```

Comment:
Only if `onSubmit` is passed as prop, wrap it in `useCallback`

Reply:
What do you mean exactly?